libraries/fiff/fiff_info.cpp

```diff
@@ -322,28 +322,28 @@ FiffInfo FiffInfo::pick_info(const RowVectorXi &sel) const
         res.chs.append(this->chs[idx]);
         res.ch_names.append(this->ch_names[idx]);
     }
-    res.nchan  = sel.size();
+    res.nchan  = static_cast<int>(sel.size());
 
     return res;
 }
 
 //=============================================================================================================
 
-QList<FiffChInfo> FiffInfo::set_current_comp(QList<FiffChInfo>& chs, fiff_int_t value)
+QList<FiffChInfo> FiffInfo::set_current_comp(QList<FiffChInfo>& _chs, fiff_int_t value)
```

Comment:
Please have a look at our [coding conventions](https://mne-cpp.github.io/pages/contribute/conv_style.html). If you want to change this parameter's name I would propose `lChs`.

Reply:
done.